core/bin/selector_generator/src/main.rs

```diff
@@ -0,0 +1,105 @@
+use std::{
+    collections::HashMap,
+    fs::{File, OpenOptions},
+    io::{self},
+};
+
+use clap::Parser;
+use glob::glob;
+use serde::{Deserialize, Serialize};
+use serde_json::{from_reader, to_writer_pretty, Value};
```

Comment:
Nit: We usually use fully qualified paths for short-names functions/macros from common crates like `serde_json`, e.g. `serde_json::from_str`/`tracing::info!`. Value is up to use, but can use a fully qualified path too.